server/devdocs/index.js

```diff
@@ -227,5 +227,10 @@ module.exports = function() {
 
 	app.use( '/devdocs/service/selectors', selectors.router );
 
+	app.use(
+		'/devdocs/storybook',
+		express.static( fspath.join( __dirname, '../../public/storybook' ) )
```

Comment:
After `server/` was moved to `client/server/` in #38190, the path will need one more `../` 🙂